.github/workflows/sync-labels.yml

```diff
@@ -0,0 +1,16 @@
+name: Sync labels
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - develop
+    paths:
+      - .github/labels.yml
+jobs:
+  call-sync:
+    uses: vector-im/element-meta/.github/workflows/sync-labels.yml@develop
+    inputs:
+      LABELS: |
+        .github/labels.yml
+    secrets:
+      ELEMENT_BOT_TOKEN: ${{ secrets.ELEMENT_BOT_TOKEN }}
```

Comment:
Yeah, true. I had left it here because if this works, we also want to use it in e.g. matrix-js-sdk and I wanted to avoid running into the copy/paste error again. But can replace it with `inherit` here if you feel strongly.

Reply:
I don't, hence the tick :)